### DISCUSSION TITLE
using channels.getParticipants

### DISCUSSION THREAD

**Discussion by marlo2222** [rubenlagus/TelegramBots#1405] [General]
Hello everyone, everything good?

Does the lib now allow the listing of channel and group participants? something seen at https://core.telegram.org/method/channels.getParticipants

**Comment by NaveenB2004** [comment]
No (yet). The link you're referring to the Telegram API or TDLib (the Telegram database library - simply the client library for Telegram users, **_not only bots_**). This Java wrapper library is for the _Telegram bots API_. The bots API is [here](https://core.telegram.org/bots/api).

PS: You can use the Telegram bots API, basically this wrapper library to identify each user event in a group/super group like join and leave (using `new_chat_members` and `left_chat_member` in [Message object](https://core.telegram.org/bots/api#message)). But you need to maintain a dataset to record each event and when you want the full list, you can get it from there.
